client/py_client/ipfs.py

```diff
@@ -1,16 +1,23 @@
 import subprocess
 import re
 import warnings
+import os
 
-ICONS_PATH = '../test-data/icons'
+local_ipfs_key = None
 
+if os.path.isfile("client\py_client\config.py"):
+    from config import loadIPFSkey
+    loadIPFSkey()
+    local_ipfs_key = os.environ['LOLCAL_IPFS_API_KEY']
+    print("local_ipfs_key: ", local_ipfs_key)
 
+ICONS_PATH = '../test-data/icons'
 class Ipfs:
     """ Minimal wrapper for the ipfs cli """
-
     @staticmethod
     def add_recursive(path_to_files):
-        ret = subprocess.run(["ipfs", "add", "-rw", path_to_files], stdout=subprocess.PIPE)
+        ret = subprocess.run(
```

Comment:
This is probably an automatic formatting issue. Can you make sure that you have linewraps at 120?

Reply:
done

Reply:
even though i just figured out to turn off totally formatting after save, which is undesired.. but I'll try to find out how to just not add new lines^^

Reply:
Haha, it's better to fix your config. I admit, googling does not yield as straightforward results, as you'd hope. Still, this page should have everything needed: https://code.visualstudio.com/docs/python/editing